.env.exampleV1AndV2.sepolia

```diff
@@ -48,6 +48,11 @@ EIGENDA_PROXY_EIGENDA_STATUS_QUERY_TIMEOUT=30m0s
 # Number of times to retry blob dispersals for V1
 EIGENDA_PROXY_EIGENDA_PUT_RETRIES=3
 
+# The EigenDA network to run on. Specifying this chooses default values for V2_DISPERSER_RPC, V2_CERT_VERIFIER_ADDR,
```

Comment:
V2_CERT_VERIFIER_ADDR isn't actually configured with this network field

Reply:
[fdd84a2](https://github.com/Layr-Labs/eigenda-proxy/pull/384/commits/fdd84a22e1cbd38cd0b82dc8675702e31e44bbdc)